### DISCUSSION TITLE
Multiple node with Tdarr LXC

### DISCUSSION THREAD

**Discussion by Narkowii** [tteck/Proxmox#3100] [Help Needed]
Hello,

Is is possible to use the Tdarr script LXC to make a simple Tdarr node to connect to another Tdarr LXC server
I have 2 pc proxmox, on the first PC i have a Tdarr server with a node, on the second i use the same install script, but i've got a 2nd server...
How to make the secon PC only a Tdarr node ?

Thanks for the help ;)

**Comment by K3v-68** [comment]
I made 3 nodes with the the lxc template and connected these, one is used as Server as well. Trying to figure out an issue of ffmpeg processes not closing on host when tdarr node is paused I came to this question: Is it actually ok to have the server process running on all nodes as well? Even is only used as Node ? This might be my problem...

**Reply by Narkowii** [reply]
Finally, it works for me with 5 nodes and 1 server.
For all LXC node only, i disable the server => i just remove the file _tdarr-server.service_ in /etc/systemd/system/

**Reply by K3v-68** [reply]
it's just what I did and my issue persists, basically I can pause the node, transcodes continue (I guess normal, till the ongoing ones are finish) but even if I stop the worker, my igpu is still fully utilized but nothing on-going is visible anymore in the UI.

just to detail a bit in case somone had the same issue:
I check with intel_gpu_top and I can still see 3 ffmpeg processes hammering the gpu.
I then killall -I -r ffmpeg and everything's back to normal (And my IO delay thanks me...)

And yes, I am using an external ffmpeg 6.0.1 (linked to this one in the Node_config file)
